### PR TITLE
MGMT-16744: KMM_MANAGED 

### PR DESCRIPTION
**Pull request by StephenJamesSmith**
BUG

D/S Docs: [DOCS] Spoke subscription example does not include KMM_MANAGED config
Jira: https://issues.redhat.com/browse/MGMT-16744?src=confmacro

Version(s): openshift-4.13, openshift-4.14, openshift-4.15, openshift-4.16

Link to docs preview: https://75907--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hardware_enablement/kmm-kernel-module-management.html#kmm-hub-running-kmm-on-the-spoke_kernel-module-management-operator

QE review: @cdvultur

### CODE REVIEW

**Comment by StephenJamesSmith** [comment]
@cdvultur This is the new PR. Please review and ACK the change.

**Review by cdvultur — APPROVED** [review]
/lgtm

**Comment by StephenJamesSmith** [comment]
/ telco

**Comment by ShaunaDiaz** [comment]
@StephenJamesSmith This needs a peer review before merge review. I do see one OCP bot style note you can fix already.

**Comment by StephenJamesSmith** [comment]
@ShaunaDiaz This is a BUG that added just several changes to existing modules and only required QE approval. I thought that, in cases like this, peer review wasn't needed. I had another one recently that I handled the same way.

**Comment by ShaunaDiaz** [comment]
@StephenJamesSmith Everything needs peer review ack, even if it's just a quick look.

**Comment by StephenJamesSmith** [comment]
@ShaunaDiaz I'm getting some mixed messages, then. Regardless, can we get this peer reviewed then, keeping in mind there were about 3 lines (minimally) changed?

**Comment by ShaunaDiaz** [comment]
@StephenJamesSmith Looks fine, just fix the OperatorHub reference.

File: modules/kmm-hub-running-kmm-on-the-spoke.adoc
Comment (StephenJamesSmith) [reply modules/kmm-hub-running-kmm-on-the-spoke.adoc:13]: changed.

**Pull request merged**
